ibctest/relayer.go

```diff
@@ -197,12 +220,24 @@ func (r *Relayer) StartRelayer(ctx context.Context, _ ibc.RelayerExecReporter, p
 	r.errCh = make(chan error, 1)
 	ctx, r.cancel = context.WithCancel(ctx)
 
-	args := append([]string{"--processor=events"}, pathNames...)
+	var args []string
+	if r.config.Processor == "" {
+		r.config.Processor = relayer.ProcessorEvents
+	}
+	args = append(args,
```

Comment:
Odd to append to an empty slice instead of assigning `args := []string{"--processor", ...}`.

Reply:
done